.storybook/config.js

```diff
@@ -13,12 +13,11 @@ import {
   F_REITH_SANS_REGULAR,
   F_REITH_SANS_ITALIC,
   F_REITH_SANS_BOLD,
-  F_REITH_SERIF_LIGHT,
```

Comment:
Could you please increment the top-level version number in `package.json` and `package-lock.json` and update the changelog?